src/work-allocation/components/task-assignment/task-assignment.component.ts

```diff
@@ -103,6 +116,34 @@ export class TaskAssignmentComponent implements OnInit {
     this.handleLocationChanged();
   }
 
+  /**
+   * Sets the logged in user id
+   */
+  public setUserId(): void {
+    const userInfoStr = this.sessionStorageService.getItem('userDetails');
+    if (userInfoStr) {
+      const userInfo: UserInfo = JSON.parse(userInfoStr);
+      this.userId = userInfo.id;
+    }
+    return undefined;
```

Comment:
If your return type is `void`, you shouldn't be returning anything. Is this statement needed or do you need to change your return type?

Reply:
Will remove this return statement